Readme.md

```diff
@@ -104,23 +98,15 @@ how to use Infogami (and its database, Infobase):
 
 - [Overview of OpenLibrary Service Architecture](https://openlibrary.org/about/architecture)
 
-## Running tests
-
-Open Library tests can be run using pytest (py.test).
-
-Inside vagrant, go to the application base directory:
-
-        cd /openlibrary
-        make test
```

Comment:
Why are these removed? I think testing instructions belong in the repo's readme (and are important!) These instructions are what I use to test, so I consider them accurate. If they change, they will change with the code, so should be version controlled with it too.

Reply:
I agree that anything dev-centric should have its primary home in git.

Reply:
Maybe we could link everything to this page and add the rest [here](https://github.com/internetarchive/openlibrary/wiki/Testing).
What do you think @tfmorris and @hornc